### ISSUE TITLE
Allow for multiple install configurations

### ISSUE THREAD

**Issue by peakwinter** [arkOScloud/arkos-install#8]
- Classic install, everything on SD card
- Optional install, boot sector to SD and data to USB drive

**Comment by elimisteve** [comment]
Does this help pave the way for full-disk encryption, since the data portion can be encrypted and the boot sector not?

**Comment by peakwinter** [comment]
Yep. I still need to confirm that that is possible but that is what this is intended for. Also to allow for more easily-accessible and dependable storage.

**Comment by elimisteve** [comment]
Cool. Somewhat relatedly, what do you think is required to make it easy to run arkOS in a VM?

**Comment by peakwinter** [comment]
I heard someone once got it working in QEMU, though I haven't personally tried it. Usually I suggest that people install Arch in a VM then install Genesis and run it on top of that, because the only things unique about arkOS as a distro (at the moment) are only beneficial if you use them on the Raspberry Pi.

If you want to run arkOS in a VM that is non-QEMU then that requires a version of arkOS for x86 or x64 architectures.
